eventually/amazons3/models.py

```diff
@@ -0,0 +1,4 @@
+"""
```

Comment:
delete this file because it is not in use

Reply:
File deleted